odw/services/data-warehouse/app/unstructured_data/extract.py

```diff
@@ -412,15 +412,15 @@ def stage_2_unstructured_to_medical(input: UnstructuredDataExtractParams, record
     
     cli_log(CliLogData(
         action="UnstructuredDataWorkflow",
-        message="DEBUG: Initializing LLM service for batch processing",
+        message="🔧 DEBUG: About to initialize LLM service for batch processing",
```

Comment:
Do we want to keep these icons? Haven't seen how they look, but I think other pipelines don't use icons in their logs.

Reply:
They were useful for debugging - but no longer needed.  I removed them.